src/ecs/rendering/systems.rs

```diff
@@ -102,14 +79,13 @@ impl<'a, 'b> SystemExt<'a, (&'b EventsLoop, PipelineBuilder, Option<DisplayConfi
         world.add_resource(cam);
         world.add_resource(AmbientColor(Rgba::from([0.01; 3])));
         world.register::<Transform>();
-        world.register::<LightComponent>();
-        world.register::<MaterialComponent>();
-        world.register::<MeshComponent>();
+        world.register::<Light>();
```

Comment:
Any reason to remove the "Component" part of the Component? It made it pretty easy to know the data we are manipulating.

Reply:
Simplicity vs expressiveness basically. Just make sure that we just need to make sure we are consistent across the engine with that to avoid confusion.

Reply:
I have a plan to remove (Mesh/Material/Light)Component wrapping. But it is not in the scope of this PR. I'll remove those changes